buildSrc/src/main/resources/gocd-docker-server/cron/cleanup-old-artifacts.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# 14 day retention
+echo "deleting the following artifact files older than 14 days:"
+find /godata/artifacts/pipelines/ -type f -ctime +14 -print -delete
```

Comment:
I feel `-mtime` is a better fit here since we are interested in the age of the artifact contents.